rendering/markdown.go

```diff
@@ -32,26 +34,33 @@ const (
 // MarkdownCommonHighlighter uses the blackfriday.MarkdownCommon setup but also includes
 // code-prettify formatting of BlockCode segments
 func MarkdownCommonHighlighter(input []byte) []byte {
-	renderer := highlightHTMLRenderer{blackfriday.HtmlRenderer(commonHTMLFlags, "", "")}
-	return blackfriday.MarkdownOptions(input, renderer, blackfriday.Options{
+	renderer := highlightHTMLRenderer{blackfriday.HtmlRenderer(commonHTMLFlags, "", ""), 0}
+	return blackfriday.MarkdownOptions(input, &renderer, blackfriday.Options{
 		Extensions: commonExtensions})
 }
 
 type highlightHTMLRenderer struct {
 	blackfriday.Renderer
+	checkboxIndex int8
 }
 
 // ListItem overrides the default ListItem render and adds support for GH-style
 // checkboxes on service side. For the contents of the list item beyond the
 // checkbox prefix, the default Html.ListItem is called.
-func (h highlightHTMLRenderer) ListItem(out *bytes.Buffer, text []byte, flags int) {
+// This adds a data-checkbox-index attribute that contains the ordinal of the
```

Comment:
As per Go convention comments for functions should always start with their name.